packages/composables/src/useOrderDetails.ts

```diff
@@ -192,7 +195,7 @@ export function useOrderDetails(orderId: string, associations?: ShopwareAssociat
   async function loadOrderDetails() {
     const orderDetailsResponse = await getOrderDetails(
       orderId,
-      deepMerge(orderAssociations, associations? associations: {}),
+      deepMerge(orderAssociations, associations ? associations : {}),
```

Comment:
`deepMerge(orderAssociations, associations),`

DeepMerge checks values

Reply:
This is only a formatter change, no logic is touched here